KeepingRecent.md

```diff
@@ -0,0 +1,41 @@
+
+# Keeping Recent on Facebook's Changes
```

Comment:
We should probably mention somewhere that the long term goal is to defork

Reply:
At this rate it's a *very* long term goal but yes I'll mention that

Reply:
I think its a much closer term goal.  -- At least for everything except macOS we really need to be working towards completely deforking.

Reply:
We still have a ton of diffs that Tom is reducing in priority order, but it probably depends on our definitions of long term. Realistically I don't think it'll happen until summer 2020 at the earliest. Hopefully I'm wrong and it's done sooner though